specification/operationalinsights/data-plane/readme.md

```diff
@@ -105,12 +105,12 @@ directive:
 ``` yaml $(python) && $(python-mode) == 'update'
 python:
   no-namespace-folders: true
-  output-folder: $(python-sdks-folder)/azure-loganalytics/azure/loganalytics
+  output-folder: $(python-sdks-folder)/operationalinsights/azure-loganalytics/azure/loganalytics
```

Comment:
"loganalytics" is the official name, operationalinsights is the RP name only (not visible to customer)

Reply:
I hear you. So far we've been standardizing on following the spec repo for service names. (I actually originally moved under `loganalytics` , before Wes called me out on it to align with the other repos )

Reply:
I oppose strongly on this one, our repo is the face to customer and should not show our internal organisation. Operationalinsights doesn't exist for customer, and we shall not show it in the repo.
Ready to discuss with Wes on it if necessary.